app/src/main/AndroidManifest.xml

```diff
@@ -43,7 +43,7 @@
         <activity
             android:name="com.igalia.wolvic.VRBrowserActivity"
             android:launchMode="singleInstance"
-            android:exported="false"
+            android:exported="true"
```

Comment:
I believe we cannot do this. The new main activity is the VersionCheck activity and is the only one that can be exported as far as I know. Note that VersionCheck copies all the parameters of the intent to the VRBrowserActivity so we don't really need to export this.